src/go/install.sh

```diff
@@ -191,15 +191,18 @@ else
 fi
 
 # Install Go tools that are isImportant && !replacedByGopls based on
-# https://github.com/golang/vscode-go/blob/v0.31.1/src/goToolsInformation.ts
+# https://github.com/golang/vscode-go/blob/v0.38.0/src/goToolsInformation.ts
```

Comment:
Can we bump the minor version? 

https://github.com/devcontainers/features/blob/5a3c0b3f2f91997a2ffb05527d671a594a41d96d/src/go/devcontainer-feature.json#L3

Reply:
I have updated the minor version as follows:

```
// src/go/devcontainer-feature.json
...
 "version": "1.2.0",
...
```